Strings/Cards.jsonc

```diff
@@ -91,25 +91,25 @@
 	"カラーと読み取り日時": {
 		"ja": "カラーと読み取り日時",
 		"en": "Card Color & Scanned Date",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"zh-Hans": "颜色及读取日期",
+		"zh-Hant": "顏色及讀取日期",
+		"zh-HK": "顏色及讀取日期"
 	},
 
 	"カラーのみ": {
 		"ja": "カラーのみ",
 		"en": "Card Color",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"zh-Hans": "卡片颜色",
+		"zh-Hant": "卡片顏色",
+		"zh-HK": "卡片顏色"
 	},
 
 	"このカード情報はアプリ内での表示等に使用されるもので、スキャンの際にカードに書き込まれることはありません。": {
 		"ja": "このカード情報はアプリ内での表示等に使用されるもので、スキャンの際にカードに書き込まれることはありません。",
-		"en": "This card's information is used only for display within the app, and is not written to the card when scanning.",
-		"zh-Hans": "这里的卡片资料只会用于应用内显示，不会于扫描时写进卡片。",
-		"zh-Hant": "這裏的卡片資料只會用於在程式內顯示，不會於掃描時寫進卡片。",
-		"zh-HK": "這裏的卡片資料只會於在程式內顯示，不會於掃描時寫入卡片。"
+		"en": "This card information is used for display within the app and is not written to the card when scanning.",
```

Comment:
I don't approve of the change to the English definition here

Reply:
Sorry. This was from an older version and I forgot to update it. I've fixed it now.